src/index.js

```diff
@@ -8,13 +8,7 @@ const mergeFolders = require('./merge-folders').merge
 function buildBundle (config) {
   return rollup.rollup(config)
     .then(function (bundle) {
-      return bundle.write({
-        format: config.format || 'es6',
```

Comment:
We are removing logic here. At maximum should be a test (or at least documentation enforcing establishing format in config).

Besides that I will admit much cleaner.

/cc @bahmutov